lib/youtube-dl.js

```diff
@@ -305,3 +305,21 @@ ytdl.getSubs = function(url, options, callback) {
     callback(null, files);
   });
 };
+
+/**
+ * @params {Boolean} descriptions
+ * @param {Function(!Error, Object)} callback
+ */
+ytdl.getExtractors = function(descriptions, options, callback) {
```

Comment:
oh you're right. sorry, for some reason I thought they were args to the command.